### PR TITLE
Add more features for iFlight_F745_AIO board.

### PR DESCRIPTION
**Pull request by Linjieqiang**

### CODE REVIEW

**Comment by mikeller** [comment]
Closing as duplicate of #417.

**Comment by Linjieqiang** [comment]
@mikeller Sorry, this is not the case. I made a mistake when I initiated it and deleted my source branch, so I made the second submission. So I hope to reopen the request and merge.

**Comment by Linjieqiang** [comment]
@mikeller Hello?

File: configs/default/IFRC-IFLIGHT_F745_AIO.config [hunk 1]
```diff
@@ -107,27 +118,56 @@ dma pin A03 0
...
+vtxtable channels 8
+vtxtable band 1 BOSCAM_A A CUSTOM  5865 5845 5825 5805 5785 5765 5745 5725
+vtxtable band 2 BOSCAM_B B CUSTOM  5733 5752 5771 5790 5809 5828 5847 5866
+vtxtable band 3 BOSCAM_E E CUSTOM  5705 5685 5665 5645 5885 5905 5925 5945
+vtxtable band 4 FATSHARK F CUSTOM  5740 5760 5780 5800 5820 5840 5860 5880
+vtxtable band 5 RACEBAND R CUSTOM  5658 5695 5732 5769 5806 5843 5880 5917
+vtxtable band 6 BAND_D   D CUSTOM  5362 5399 5436 5473 5510 5547 5584 5621
+vtxtable powerlevels 5
+vtxtable powervalues 25 100 200 400 600
+vtxtable powerlabels 25 100 200 400 600
```
Comment (etracer65) [inline configs/default/IFRC-IFLIGHT_F745_AIO.config:154]: Unless you are willing to certify that these settings are legal worldwide (and provide indemnity to anybody using this board), then you cannot include the `vtxtable` in the unified target. And clearly these settings **are not legal worldwide.**

**Pull request closed without merging**